src/direflow-components/App.js

```diff
@@ -1,63 +1,96 @@
+    requestEntities(props.url).then((entities) => {
       if (entities) {
-        setEntities(entities);
+        let filterEmbeddedEntity = [];
+        for (let i = 0; i < entities.length; i++) {
+          if (entities[i].queryAll) {
+            filterEmbeddedEntity.push(entities[i]);
+          }
+        }
+        setEntities(filterEmbeddedEntity);
       }
     });
   }, [props.url]);
 
   const dispatch = useContext(EventContext);
 
-  const handleClick = () => {
-    const event = new Event('my-event');
-    dispatch(event);
+  const handleClick = (entity, ind) => {
+    setCurrentEntity(entity);
+    setResultTitle(`Resultados de ${entity.name}`);
+    setSelectedKey(ind.toString());
   };
-
-  const renderSampleList = props.sampleList.map((sample) => (
-    <div key={sample} className='sample-text'>
-      → {sample}
-    </div>
+  const renderEntities = entities.map((entity, index) => (
+    <Menu.Item key={index} onClick={() => handleClick(entity, index)}>
+      <Paragraph
+        style={{
+          color: "white",
+          textTransform: "uppercase",
+          fontWeight: "bold",
+        }}
+      >
+        {entity.name}
+      </Paragraph>
+    </Menu.Item>
   ));
 
+  const onCollapse = (collapsed) => {
+    setCollapsed(collapsed);
+  };
   return (
     <EntitiesContext.Provider value={entities}>
       <Styled styles={styles}>
-        <div className='app'>
-          <div className='top'>
-            <div className='header-image' />
-          </div>
-          <div className='bottom'>
-            <div className='header-title'>{props.componentTitle}</div>
-            <div>{renderSampleList}</div>
-            <button className='button' onClick={handleClick}>
-              Click me!
-            </button>
-          </div>
-        </div>
+        <Layout style={{ minHeight: "100vh", display: "flex", flex: "auto" }}>
+          <Sider collapsible collapsed={collapsed} onCollapse={onCollapse}>
+            <Menu theme="dark" selectedKeys={[selectedKey]} mode="inline">
+              {renderEntities}
+            </Menu>
+          </Sider>
+          <Layout className="site-layout">
+            <Header className="site-layout-background" style={{ padding: 0 }} />
+            <Content style={{ margin: "0 16px" }}>
+              <Title level={2} style={{ textAlign: "center" }}>
+                {resultTitle}
+              </Title>
+              <Table displayEntities={currentEntity} />
+            </Content>
+            <Footer style={{ textAlign: "center" }}>Simtlix ©2021</Footer>
+          </Layout>
+        </Layout>
       </Styled>
     </EntitiesContext.Provider>
   );
 };
 
 App.defaultProps = {
-  url: '',
-  componentTitle: 'Simfinity Web',
+  url: "",
+  componentTitle: "Simfinity Web",
   sampleList: [
-    'Create with React',
-    'Build as Web Component',
-    'Use it anywhere!',
+    "Create with React",
+    "Build as Web Component",
+    "Use it anywhere!",
   ],
-}
+};
```

Comment:
I think we can remove 'componentTitle' and 'sampleList' from defaultProps and from propTypes below